### DISCUSSION TITLE
Error: Variable "$id" of required type "Int!" was not provided.

### DISCUSSION THREAD

**Discussion by arkansas-coder** [requarks/wiki#6203] [Error / Bug Report] [answered]
I have a page whose location is `To-Do`. The system accepted it without error, but when I try to do anything with the page, I get an OOPs! error. When I try to delete it, I get the above error. Are hyphens (`-`) disallowed in a location name? If so, the system should not accept a name containing it. If it is allowed, there is another error type. What information can I provide to track this down? How do I delete this page?

**Comment by NGPixel** [comment, marked as answer]
It shouldn't have allowed you to create it in the first place.
Paths in the format `aa` or `aa-aa` at the root are not allowed because it is reserved for locales (e.g. `en` or `fr-ca`).

See https://docs.requarks.io/guide/pages#naming-restrictions

**Comment by arkansas-coder** [comment]
The documentation does not say that this is disallowed. It only says that IETF Language tags are disallowed, but 'To-Do' does not appear to be an IETF tag. Given that the developers appear to have meant that the format rather than actual IETF tags is what should have been disallowed, it is a system (input validation) bug. 

Notwithstanding, how can I delete the page?

Thanks
